ssz/src/main/java/tech/pegasys/artemis/ssz/backing/view/ListViewWriteImpl.java

```diff
@@ -36,17 +34,18 @@
       extends ContainerViewWriteImpl {
     private final VectorViewType<ElementReadType> vectorType;
 
-    public ListContainerWrite(ListContainerRead<ElementReadType> backingImmutableView) {
+    public ListContainerWrite(
+        ListViewReadImpl.ListContainerRead<ElementReadType> backingImmutableView) {
```

Comment:
nit: Import `ListContainerRead` and `UInt64View` directly.

Reply:
Done.